### PR TITLE
Updating copy

### PR DESCRIPTION
**Pull request by Duncid**
## Description

Updating copy of modals validation for changing sharing status of an assistant.

### CODE REVIEW

File: front/components/assistant_builder/TeamSharingSection.tsx [hunk 1]
```diff
@@ -105,7 +105,7 @@ export function TeamSharingSection({
       text: "Only I can view and edit.",
       confirmationModalData: {
         title: "Moving to Personal Assistants",
-        text: `Moving the assistant to your Personal Assistants will make the assistant unaccessible to other members of the workspace.`,
+        text: `The assistant is only editable, viewable and usable to you.`,
```
Comment (philipperolet) [inline front/components/assistant_builder/TeamSharingSection.tsx:108]:
```suggestion
        text: `The assistant is only editable, viewable and usable by you.`,
```

File: front/components/assistant_builder/TeamSharingSection.tsx [hunk 2]
```diff
@@ -118,7 +118,7 @@ export function TeamSharingSection({
     title: "Moving to Shared Assistants",
     text: `Moving ${
       assistantName || "the assistant"
-    } to Shared Assistants will make the assistant editable by all members of the workspace and the assistant will not be activated by default anymore.`,
+    } will make it editable by all members and automatically appears in every member's 'My Assistants' list anymore.`,
```
Comment (philipperolet) [inline front/components/assistant_builder/TeamSharingSection.tsx:121]:
```suggestion
    } will make it editable by all members; it won't automatically appear in every member's 'My Assistants' list anymore.`,
```

**Review by philipperolet — COMMENTED** [review]
2 nits otherwise LGTM :+1:

**Comment by Duncid** [comment]
Thank you for the suggestions @philipperolet !
I added them.
Merging when tests are done.

**Pull request merged**